README-KO.md

```diff
@@ -189,7 +189,8 @@ Scala는 매우 강력하며 여러가지 페러다임에 적용 가능한 언
   }
   ```
 
-- 함수 선언에서 파라메터가 한 줄에 맞지 않아 들여쓰기를 하는 경우, 4칸 공백을 합니다. 반환 타입은 다음 줄에 배치되거나 같은 라인에 배치될 수 있습니다. 다음 라인에 쓰는 경우, 2칸 들여쓰기를 합니다.
+- 함수 선언에서 파라메터가 두 줄에 맞지 않아 들여쓰기를 하는 경우, 각 인자에 4칸 공백을 사용하고 각 라인에 배치 합니다. 반환 타입은 다음 줄에 배치되거나 같은 라인에 배치될 수 있습니다. 다음 라인에 쓰는 경우, 2칸 들여쓰기를 합니다.
```

Comment:
In case of Korean, I believe it's find as is for https://github.com/databricks/scala-style-guide/pull/64#discussion_r192308713 and https://github.com/databricks/scala-style-guide/pull/64#discussion_r192308954.